composer.json

```diff
@@ -50,6 +54,7 @@
 		"typo3/cms-indexed-search": "dev-master as 11.3.1",
 		"typo3/cms-info": "dev-master as 11.3.1",
 		"typo3/cms-install": "dev-master as 11.3.1",
+		"typo3/cms-introduction": "dev-master",
```

Comment:
composer.json's 
```
"typo3/cms-introduction": "dev-master"
```
does not match with composer.lock's
```
"typo3/cms-introduction": "dev-BP_12-0"
```

Reply:
I think you are mixing up something here, cms-introduction ist also as dev-master in the composer.lock

It is the bootstrap package we need as 
            "name": "bk2k/bootstrap-package",
            "version": "dev-BP_12-0",